Makefile

```diff
@@ -102,8 +102,10 @@ dep-validate: dep-restore
 	@echo "+ $@"
 	@rm -Rf .vendor.bak
 	@mv vendor .vendor.bak
+	@mv Godeps/Godeps.json .Godeps.json.bak
 	@rm -Rf Godeps
-	@$(GODEP) save ./...
+	@$(GODEP) save $(PKGS)
 	@test -z "$$(diff -r vendor .vendor.bak 2>&1 | tee /dev/stderr)" || \
 		(echo >&2 "+ borked dependencies! what you have in Godeps/Godeps.json does not match with what you have in vendor" && false)
 	@rm -Rf .vendor.bak
+	@mv .Godeps.json.bak Godeps/Godeps.json
```

Comment:
Doesn't this task generates a new `Godeps.json` file? (eg: which will make the move fail)